<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QlineEdit,
QlistView{
	color:rgb(127,0,63);
	background-color:rgb(255,255,241);
	selection-color:white;
	selection-background-color:rgb(191,31,127);
	border: 2px groove gray;
	border-radius: 10px;
	padding: 2px 4px;
}

background-color: transparent;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{border-radius: 30px;border-image: url(:/new/prefix1/bei jin 3.png);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>131</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{border: 2px groove blue;border-radius: 8px;}</string>
   </property>
   <item>
    <property name="text">
     <string>暗少科</string>
    </property>
    <property name="font">
     <font>
      <family>华文隶书</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>傻扬科</string>
    </property>
    <property name="font">
     <font>
      <family>华文隶书</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>暗黑科</string>
    </property>
    <property name="font">
     <font>
      <family>华文隶书</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黑少科</string>
    </property>
    <property name="font">
     <font>
      <family>华文隶书</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>皮肤科</string>
    </property>
    <property name="font">
     <font>
      <family>华文隶书</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/new/prefix1/xie1.png);}



QPushButton:hover{border-image: url(:/new/prefix1/xie2.png);}

QPushButton:pressed{border-image: url(:/new/prefix1/xie3.png);}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{border: 2px groove blue;border-radius: 4px;}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>医保卡号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{border: 2px groove blue;border-radius: 4px;}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文隶书</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>电话</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文隶书</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>年龄</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{border: 2px groove blue;border-radius: 4px;}</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>261</width>
     <height>381</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/doctor6副本.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>170</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/prefix1/提交2.png);}
QPushButton:hover{border-image: url(:/new/prefix1/提交1.png);}
QPushButton:pressed{border-image: url(:/new/prefix1/提交3.png);}
/*QPushButton{border-image: url(:/new/prefix1/挂号图标副本2.png);}
QPushButton:hover{border-image: url(:/new/prefix1/挂号图标副本4.png);}

	
QPushButton:pressed{border-image: url(:/new/prefix1/挂号图标副本3.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{border: 2px groove blue;border-radius: 4px;}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{border: 2px groove blue;border-radius: 4px;}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文隶书</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文隶书</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
